app/src/frontend/building/sidebar.css

```diff
@@ -9,13 +9,6 @@
     height: 40%;
 }
 
-.info-container h2:first-child {
-    margin-bottom: 0.5rem;
-    margin-top: 0.5rem;
-    margin-left: -0.1em;
-    padding: 0 0.75rem;
-}
-
```

Comment:
I couldn't see any elements in the current code that this style would affect, but it did affect the new edit history-related elements. I removed to not have to play with selector specificity or `!important` rules.